src/main/java/io/phasetwo/keycloak/magic/auth/MagicLinkContinuationAuthenticator.java

```diff
@@ -31,6 +32,8 @@
 @JBossLog
 public class MagicLinkContinuationAuthenticator extends UsernamePasswordForm {
 
+  static final String CREATE_NONEXISTENT_USER_CONFIG_PROPERTY = "ext-magic-create-continuation-nonexistent-user";
```

Comment:
Probably a good time to make this consistent. Move the prop keys to the `MagicLink` class and reuse the naming in both places.

Reply:
Added it to the `MagicLink` class, replaced all usages.